infra/gcp/clusters/k8s-infra-prow-build-trusted/prow-build-trusted/main.tf

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2020 The Kubernetes Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+This file defines:
+- Google Project k8s-infra-prow-build-trusted to host the cluster
+- GCP Service Account for prow-build-trusted
+- GKE cluster configuration for prow-build-trusted
+- GKE nodepool configuration for prow-build-trusted
+*/
+
+locals {
+  project_id              = "k8s-infra-prow-build-trusted"
+  cluster_name            = "prow-build-trusted"   // The name of the cluster defined in this file
+  cluster_ksa_name        = "prow-build-trusted"   // MUST match the name of the KSA intended to use the prow_build_cluster_sa serviceaccount
+  cluster_location        = "us-central1"          // The GCP location (region or zone) where the cluster should be created
+  bigquery_location       = "US"                   // The bigquery specific location where the dataset should be created
+  pod_namespace           = "test-pods"            // MUST match whatever prow is configured to use when it schedules to this cluster
+}
+
+// TODO: I think more people than me should have owner/edit access to this project
```

Comment:
?

Reply:
dropped this, my concerns are better explained in https://github.com/kubernetes/k8s.io/issues/844 which I'll use to track followup